### PR TITLE
Fix #7352: Get rid of most duplicate copies of Ahem

### LINKED ISSUE #7321: ./css-font-display/resources/slow-ahem-loading.py should just use trickle
See https://wptserve.readthedocs.io/en/latest/pipes.html#trickle

Note #7303 may move this file.

### LINKED ISSUE #7352: Different versions of Ahem.ttf throughout the repo
I notice that we have multiple copies of `[A|a]hem.ttf` in the repo:

```
find . -name "?hem.ttf"|xargs sha1sum
8cdc9e68594fbb6db8c7b4bff643ab2432b51db6  ./content-security-policy/support/Ahem.ttf
88d834aadcd7b547720992a2894b9ab3251e3e04  ./css/fonts/ahem/ahem.ttf
8cdc9e68594fbb6db8c7b4bff643ab2432b51db6  ./css/vendor-imports/mozilla/mozilla-central-reftests/ruby/support/Ahem.ttf
8cdc9e68594fbb6db8c7b4bff643ab2432b51db6  ./css/vendor-imports/mozilla/mozilla-central-reftests/text3/support/Ahem.ttf
8cdc9e68594fbb6db8c7b4bff643ab2432b51db6  ./css/vendor-imports/mozilla/mozilla-central-reftests/flexbox/support/Ahem.ttf
8cdc9e68594fbb6db8c7b4bff643ab2432b51db6  ./css/vendor-imports/mozilla/mozilla-central-reftests/variables/support/Ahem.ttf
8cdc9e68594fbb6db8c7b4bff643ab2432b51db6  ./fonts/Ahem.ttf
```

`./css/fonts/ahem/ahem.ttf` is the newest version 1.2, while everything else is version 1.1. The two versions don't exactly match (compared by using the `assumptions/ahem.html` reftest).

### PR DESCRIPTION
**Pull request by gsnedders**
This PR leaves us with just `fonts/Ahem.ttf` and `css/vendor-imports/mozilla/mozilla-central-reftests/*/support/Ahem.ttf`, and the latter of those is maintained in mozilla-central and one-way sync'd to here (so any attempt to get rid of it here will just be overwritten).

Fixes #7321, fixes #7352.

### CODE REVIEW

**Comment by gsnedders** [comment]
@lithium is https://gist.github.com/gsnedders/d046dd8b41d0f5338696f85ee052fb2b#file-css-fonts-ahem-ahem-ttf-diff what we expect to change v. the last copy in css/fonts/ahem/ahem.ttf? Also https://github.com/w3c/web-platform-tests/blob/d3eef9f6750167147761ff6f39445e0d8748d736/css/fonts/ahem/README seems to have more credits than ahem.ttf contains?

File: assumptions/tools/ahem-generate-table.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def grouper(n, iterable):
     iterable = iter(iterable)
     return iter(lambda: list(itertools.islice(iterable, n)), [])
 
-ttf = TTFont("../../css/fonts/ahem/ahem.ttf")
+ttf = TTFont("../../fonts/Ahem.ttf")
```
Comment (jgraham) [inline assumptions/tools/ahem-generate-table.py]: I would prefer absolute paths if possible.

**Review by jgraham — APPROVED** [review]
Would ideally like a lint to stop people adding more copies of this in the future.

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/18613)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/276888362)

File: assumptions/tools/ahem-generate-table.py (hunk 1 above)
Comment (gsnedders) [reply assumptions/tools/ahem-generate-table.py]: How do we get an absolute path to the root of the repo?

**Comment by gsnedders** [comment]
Uh, @litherum https://github.com/w3c/web-platform-tests/pull/7372#issuecomment-330250872 above should've been directed at you.

File: assumptions/tools/ahem-generate-table.py (hunk 1 above)
Comment (jgraham) [reply assumptions/tools/ahem-generate-table.py]: `/fonts/Ahem.ttf`?

**Review by jgraham — APPROVED** [review]

File: assumptions/tools/ahem-generate-table.py (hunk 1 above)
Comment (gsnedders) [reply assumptions/tools/ahem-generate-table.py]: That won't work. This is a Python script. That's resolved relative to the filesystem root and not the repo root. As you said in IRC, you're "being dumb". Ignoring and merging. :)

**Pull request merged**